public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -243,9 +258,10 @@ export function RichHistoryCard(props: Props) {
       <div className={cx(styles.cardRow)}>
         <div className={styles.queryContainer}>
           {query.queries.map((q, i) => {
+            const queryText = createQueryText(q, queryDsInstance);
             return (
               <div aria-label="Query text" key={`${q}-${i}`} className={styles.queryRow}>
-                {q}
+                {queryText.length > 0 && queryText}
```

Comment:
If `queryText` is empty string we render false here which ends up as nothing right? is that different from just rendering empty text?

Reply:
true, removed.